qiskit/extensions/unitary.py

```diff
@@ -14,7 +14,6 @@
 Arbitrary unitary circuit instruction.
 """
 
-from collections import OrderedDict
```

Comment:
This seems to be  a completely unrelated change - I guess the import is not used and its being fixed by this too

Reply:
I think that'll disappear once we merge from main, where this has been fixed independently